### PR TITLE
Add new test and fix for float configuration failure in conf_remap

### PR DESCRIPTION
**Pull request by willwendorfvm**
Test that conf_remap can override float settings (e.g. proxy.config.http.background_fill_completed_threshold) successfully.

### CODE REVIEW

**Comment by randall** [comment]
[approve ci]

**Comment by willwendorfvm** [comment]
Just fixed a typo, forgot to save the file I changed (got rid of a mysterio name)

File: tests/gold_tests/remap/conf_remap_float.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# First reload
+tr = Test.AddTestRun("traffic_ctl command")
+tr.Env = ts.Env
+tr.TimeOut = 5
+tr.StillRunningAfter = ts
+
+p = tr.Processes.Default
+p.Command = "traffic_ctl config describe proxy.config.http.background_fill_completed_threshold"
+# p.Command = "curl http://127.0.0.1:{0}".format(ts.Variables.port)
```
Comment (randall) [inline tests/gold_tests/remap/conf_remap_float.py:50]: small nit, can you remove the commented out code here?

Comment (willwendorfvm) [reply tests/gold_tests/remap/conf_remap_float.py]: Done.

**Comment by randall** [comment]
[approve ci]

**Comment by randall** [comment]
Sorry, one more nit, can you change the subject of the actual commit to the same thing as the PR? Eg `Add new test and fix for float configuration failure in conf_remap`

the current subject `Add new test and fix for 3309` is not that useful in the git history context.

**Comment by willwendorfvm** [comment]
Amended commit message as requested.

**Comment by randall** [comment]
[approve ci]

**Pull request merged**